### ISSUE TITLE
other strategies

### ISSUE THREAD

**Issue by andrewczgithub** [CoinAlpha/hummingbot#672]
hi all
love the project.
Will other strategies be added like trend following, machine learning prediction.
The bot would look amazing running some of these standard traditional future trading
strategies.
Best,
Andrew

**Comment by mifeng** [comment]
YES. Our company actually started as an ML-based momentum / mean reversion quant hedge fund, so this is definitely on our roadmap. We've focused on writing market making and arbitrage type strategies thus far because those strategies are easier for users without much data science experience to run and make profits, but we're highly interested in creating directional strategies that use TA based indicators as well.

If you know of folks with Python chops who may be interested in creating such a strategy, please let me know!

**Comment by asmodehn** [comment]
Just a pointer for TA in python that I hope will be useful : https://github.com/twopirllc/pandas-ta
As a side note, having a pandas dataframe based interface to OHLCV data to the script, so users can write their strategy in python seems a sensible choice to me...
